client/lib/abtest/active-tests.js

```diff
@@ -88,16 +88,6 @@ export default {
 		},
 		defaultVariation: 'control',
 	},
-	domainManagementSuggestionV2: {
-		datestamp: '20181001',
-		variations: {
-			domainsbot_front: 80,
-			variation_front: 20,
-		},
-		defaultVariation: 'domainsbot_front',
-		assignmentMethod: 'userId',
```

Comment:
Are you sure `assignmentMethod: 'userId',` is not needed anymore? Looks like this method would be useful for this case (since this is only in domain management, so user should always be logged in).

Reply:
Good catch. I meant to include this.